### ISSUE TITLE
MultiDex problem

### ISSUE THREAD

**Issue by HectorFGarcia** [ZieIony/Carbon#132]
When I add this two lines:

renderscriptTargetApi 20
renderscriptSupportModeEnabled true

I'm having this problem:

Error:Execution failed for task ':app:packageAllDebugClassesForMultiDex'.

> java.util.zip.ZipException: duplicate entry: android/support/annotation/Keep.class

What can I do to prevent that. I can't turn multiDex off.

**Comment by ZieIony** [comment]
Are you, by any chance, using support lib v23? This error is widely seen after updating the support lib from v22 to v23.

**Comment by HectorFGarcia** [comment]
Yes, exactly. Keep.class is in annotation-support v23 lib and the render v8 lib too.

**Comment by ZieIony** [comment]
As for now I don't exactly have any way of dealing with that issue, I'm closely watching the StackOverflow thread of your issue. Pasting here for others: http://stackoverflow.com/questions/32327227/duplicate-class-in-two-offcial-android-libs-gradle-fails

**Comment by HectorFGarcia** [comment]
Has you reported this to Google? If the bug comes from two of their libraries, they have to fix it.

By the way, excuse my english.

**Comment by ZieIony** [comment]
Don't worry, it's pretty good :)

No, but someone did. There are at least two reports:
https://code.google.com/p/android/issues/detail?id=179493
https://code.google.com/p/android/issues/detail?id=181697

**Comment by HectorFGarcia** [comment]
Ok. I hope they release an update in a few weeks. Until now I'll keep working with API 22 and checking this.

**Comment by HectorFGarcia** [comment]
The 23.0.1 support library doesn't fix this yet?

**Comment by ZieIony** [comment]
The bug report on gogle tracker is still open. I've updated build script, so Carbon now uses the most recent libs and SDK. It works fine in all projects I'm working on, but maybe I'm not hitting that broken configuration.

**Comment by HectorFGarcia** [comment]
Error:Execution failed for task ':app:packageAllDebugClassesForMultiDex'.

> java.util.zip.ZipException: duplicate entry: android/support/annotation/LayoutRes.class

:_

**Comment by ZieIony** [comment]
```
#11 efren.fe...@gmail.com 
Seems to be fixed in build tools 23.0.1 :)
```

**Comment by ZieIony** [comment]
Even if it's not fixed, I'm closing the issue as there's nothing I can do about it.
